OWNERS

```diff
@@ -0,0 +1,11 @@
+# approval == this is a good idea /approve
+approvers:
```

Comment:
The github IDs are not sufficient, we should at least have the format to include,
<Full name>, <githubID> and optionally <mailID> ([example](https://github.com/rook/rook/blob/master/OWNERS.md))

If we are thinking of using it for future automation like `https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md` so, then we may need to just leave github IDs, but even then the format can include required details as mentioned and tooling can pick up the github IDs as required.